android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -379,6 +379,10 @@ public void call(DownloadProgress downloadProgress) {
                     } catch (IOException e) {
                         e.printStackTrace();
                         promise.reject(e.getMessage());
+                    } catch (CodePushInvalidUpdateException e) {
+                        e.printStackTrace();
+                        saveFailedUpdate(updatePackage);
+                        promise.reject(e.getMessage());
```

Comment:
It currently says this: "Update is invalid - no files with extension .bundle, .js or .jsbundle were found in the update package."